scalableminds/webknossos
# add publication and details to dataset (backend only)

### URL of deployed dev instance (used for testing):
- https://publications.webknossos.xyz

### Steps to test:
- add details and publication to a dataset in your database, e.g.
- `insert into webknossos.publications(_id, publicationDate, imageUrl, description) values('5c3c9ec895010095014759fd',NOW(),'https://webknossos.org/images/oxalis.svg','This is a wonderful dummy publication, it has authors, it has a link, it has a doi number, those could go here');`
- `update webknossos.datasets set _publication = '5c3c9ec895010095014759fd', details='{"species":"Drosophila melanogaster"}' where _id = '<A-DATASET-ID>' ;` (edit dataset id!)
- dataset list request should contain matching json values

### Issues:
- backend part for #3515 

------
